sdk/nodejs/provisioning/engineconn.ts

```diff
@@ -0,0 +1,27 @@
+import Client from "../api/client.js";
+import { StdioOption } from 'execa';
+
+export interface ConnectOpts {
+  Workdir?: string;
+  Project?: string;
+  OutputLog?: StdioOption;
```

Comment:
Another follow up fix, we should probably not use an `execa` type in our public interface since we may want to switch to a different library in the future. I think the nodejs `Stream` type might be all we need.

Reply:
issue https://github.com/dagger/dagger/issues/3836 created